example/mapboxExample.js

```diff
@@ -0,0 +1,145 @@
+import * as THREE from 'three';
+import CameraSync, { projectToWorld, projectedUnitsPerMeter } from './mapboxExampleCamera.js';
+import { DebugTilesRenderer as TilesRenderer } from '../src/index.js';
+
+function rotationBetweenDirections( dir1, dir2 ) {
+
+	const rotation = new THREE.Quaternion();
+	const a = new THREE.Vector3().crossVectors( dir1, dir2 );
+	rotation.x = a.x;
+	rotation.y = a.y;
+	rotation.z = a.z;
+	rotation.w = 1 + dir1.clone().dot( dir2 );
+	rotation.normalize();
+
+	return rotation;
+
+}
+
+const origin = [ 116, 35 ];
+
+// TO MAKE THE MAP APPEAR YOU MUST
+// ADD YOUR ACCESS TOKEN FROM
+// https://account.mapbox.com
+// This is my private key, it is prohibited to use in the project
+window.mapboxgl.accessToken =
+        'pk.eyJ1IjoieGlheGlhbmdmbmVnIiwiYSI6ImNscTRzc245NTA5c3cya3BhdXA4MDY0NWQifQ._9T_J9z7Hvcd00nQL3UuSw';
+const map = new window.mapboxgl.Map( {
```

Comment:
This is a key created by my own account, which can be used normally. I just don't want anyone to use my key in their project. If someone uses it, they will have to bear the risk of failure themselves

Reply:
This comment can be deleted

// This is my private key, it is promoted to use in the project

Reply:
I think it's best to use the same model as the other examples for the moment and require that users provide their own key in the GUI to use the demo. Otherwise it's difficult to track who is responsible for which key and where they've come from.

Reply:
ok